docs/build.sh

```diff
@@ -36,12 +38,8 @@ mkdir -p "${DOCS_OUTPUT_DIR}"
 rm -rf "${GENERATED_RST_DIR}"
 mkdir -p "${GENERATED_RST_DIR}"
 
-if [ ! -d "${BUILD_DIR}"/venv ]; then
-  virtualenv "${BUILD_DIR}"/venv --no-site-packages --python=python2.7
-  "${BUILD_DIR}"/venv/bin/pip install -r "${SCRIPT_DIR}"/requirements.txt
-fi
-
-source "${BUILD_DIR}"/venv/bin/activate
+source_venv $BUILD_DIR
```

Comment:
Nit: `"${BUILD_DIR}"` (and fix other unquoted use of directories below for whitespace safety)